packages/linea-ccip-gateway/package.json

```diff
@@ -1,56 +1,59 @@
 {
   "name": "@consensys/linea-ccip-gateway",
   "version": "1.1.0",
-  "main": "dist/index.js",
-  "author": "Consensys",
   "license": "Apache-2.0",
+  "author": "Consensys",
+  "main": "dist/index.js",
   "scripts": {
-    "start": "node dist/server.js",
-    "build": "tsc --project tsconfig.json --module commonjs --skipLibCheck true --outDir ./dist",
+    "build": "tsc --project tsconfig.build.json",
     "clean": "rm -fr node_modules dist",
-    "test": "hardhat compile && mocha test/testVerifier.spec.ts --timeout 20000 --exit",
-    "compile": "hardhat compile"
+    "compile": "hardhat compile",
+    "lint": "eslint . --ext ts --report-unused-disable-directives --max-warnings 0",
+    "prettier": "prettier --check '**/*.ts'",
+    "prettier:fix": "prettier --write '**/*.ts'",
+    "start": "node dist/server.js",
+    "test": "hardhat compile && mocha test/testVerifier.spec.ts --timeout 20000 --exit"
   },
   "dependencies": {
+    "@chainlink/ccip-read-server": "^0.2.1",
+    "@consensys/linea-state-verifier": "^1.0.1",
     "@ethersproject/abi": "^5.4.7",
     "@ethersproject/bytes": "^5.7.0",
-    "@chainlink/ccip-read-server": "^0.2.1",
     "commander": "^11.0.0",
     "cors": "^2.8.5",
     "dotenv": "^16.0.3",
     "ethers": "^6.11.1",
     "express": "^4.19.2",
-    "@consensys/linea-state-verifier": "^1.0.1",
     "tsdx": "^0.14.1",
     "tslib": "^2.6.2",
-    "winston": "^3.9.0",
-    "typescript": "^5.4.5"
+    "typescript": "^5.4.5",
+    "winston": "^3.9.0"
```

Comment:
while you are here, I would specify/fix the versions

Reply:
Still, I upgraded a few dependencies to improve the test environment.